Libplanet/Blocks/Block.cs

```diff
@@ -173,7 +173,8 @@ public byte[] ToBencodex(bool hash, bool transactionData)
                 );
             foreach (Transaction<T> tx in Transactions)
             {
-                delta = tx.EvaluateActions(Hash, Index, delta);
+                delta = tx.EvaluateActions(
+                    Hash, Index, delta, RewardBeneficiary.Value);
```

Comment:
It seems that `rewardBeneficiary` is no longer a proper name anymore, how about fixing it this time?

Reply:
I'm for this proposal.  How about `Block<T>.Miner`?

Reply:
I agree. `Block<T>.Miner` seems good.